src/Form/HmNewsletterNewsletterAdminForm.php

```diff
@@ -82,6 +89,14 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#default_value' => $hm_newsletter_settings->get('hm_imprint_text'),
     );
 
+    $displayed_agreements = implode('|', $hm_newsletter_settings->get('hm_displayed_agreements'));
```

Comment:
This throws an "implode(): Invalid arguments" warnig, when module was updated and the configuration edit screen is used